src/Symfony/Component/Validator/Resources/translations/validators.hu.xlf

```diff
@@ -366,6 +366,22 @@
                 <source>This value should be between {{ min }} and {{ max }}.</source>
                 <target>Ennek az értéknek {{ min }} és {{ max }} között kell lennie.</target>
             </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target>Ez az érték nem érvényes állomásnév (hosztnév).</target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target>A gyűjetemény elemszámának oszthatónak kell lennie a következővel: {{ compared_value }}.</target>
```

Comment:
A typo here, it should be "gyűjtemény". Anyway, how about starting this way: "A gyűjteményben lévő elemek számának..."

Reply:
Thanks, it's better this way.